onnxscript/tests/function_libs/torch_aten/ops_correctness_test.py

```diff
@@ -97,16 +101,18 @@ def skip(
     reason: str,
     dtypes: Optional[Collection[torch.dtype]] = None,
     matcher: Optional[Callable[[Any], Any]] = None,
+    active_if: bool = True,
 ) -> DecorateMeta:
     """Skips an OpInfo test.
 
     Args:
         op_name: The name of the operator.
         variant_name: Optional OpInfo variant_test_name.
-        dtypes: The dtypes to skip.
         reason: The reason for skipping.
+        dtypes: The dtypes to skip.
         matcher: A function that matches the test sample input. It is used only when
             xfail is in the SKIP_SUBTESTS list.
+        active_if: Whether the skip is active.
```

Comment:
A little confused: I see this parameter was used in xfail() function, while this description is saying about skip.

Is this a general parameter, or related to skip or xfail?

Reply:
Sorry that was a typo. It’s general and inherited from PyTorch

Reply:
Updated. I didn't create the name but it makes sense: when used it becomes something like `active_if=onnxruntime_older_than("1.14"),`, which is good for redability.

Reply:
Nit:
I agree that "active_if" also makes sense. 

I'm thinking this in terms of unified style of our code. While I go through our code base, we didn't use "active" to describe if something will work or not. Most of time, we will use "enabled" for such purpose, like cudnn_enabled, logger.isEnabledFor. 

So probably "enabled_if" would be better.

Reply:
SG!

Reply:
Done